js/render/draw_background.js

```diff
@@ -3,7 +3,7 @@
 var TilePyramid = require('../source/tile_pyramid');
 var pyramid = new TilePyramid({ tileSize: 512 });
 var util = require('../util/util');
-var EXTENT = require('../data/bucket').EXTENT;
+var extent = 8192;
```

Comment:
Why did you change this line? It is important that we keep a single canonical definition of `EXTENT`.